cmd/wampproto/helpers.go

```diff
@@ -0,0 +1,16 @@
+package main
```

Comment:
should be moved out of the cmd package to the root of this repository. Create helpers.go at the root of the project